app/controllers/admin/customers_controller.rb

```diff
@@ -74,7 +74,12 @@ def destroy
     def collection
       if json_request? && params[:enterprise_id].present?
         customers_relation.
-          includes(:bill_address, :ship_address, user: :credit_cards)
+          includes(
+            :enterprise,
+            { bill_address: [:state, :country] },
+            { ship_address: [:state, :country] },
```

Comment:
this is particular eager loading is becoming a pattern :thinking: perhaps hunting all its occurrences could be a quick way to get a cheap perf boost across the app.